### DISCUSSION TITLE
Institution name and domain address @ibch

### DISCUSSION THREAD

**Discussion by Julentsia** [RosettaCommons/rosetta#91] [ROSIE]
Dear ROSIE Support Team,

I am writing to inform you that my email address with the domain "@ibch.ru" belongs to a research institute called "IBCh RAS" (Shemyakin-Ovchinnikov Institute of Bioorganic Chemistry of the Russian Academy of Sciences), which is located in Moscow, Russia. 

I would appreciate it if you could add my email address to your system to grant me access to the server.
Thank you for your attention to this matter.

Best regards,
Yulia A. Logashina, PhD
research fellow of neuroreceptors & neuroregulators laboratory
IBCh RAS

**Comment by roccomoretti** [comment]
Unfortunately, due to the ongoing situation in eastern Europe, there are legal/administrative issues with using servers in the United States to provide computational services to people in Russia, which means that we are unable to approve Russian email addresses for ROSIE accounts at this time.

We apologize for the current state of the world.
